src/components/Calendarweek.tsx

```diff
@@ -0,0 +1,35 @@
+import React,{ useEffect } from 'react';
+import FullCalendar from '@fullcalendar/react'
+import dayGridPlugin from '@fullcalendar/daygrid'
+import interactionPlugin from '@fullcalendar/interaction'
+import timeGridPlugin from "@fullcalendar/timegrid";
+
+class Calendarweek extends React.Component {
```

Comment:
`CalendarWeek`, or better: `WeeklyCalendar`.

Reply:
I changed the file names in new commits